seatunnel-engine/seatunnel-engine-client/src/main/java/org/apache/seatunnel/engine/client/LocalExecutionContext.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.engine.client;
+
+import org.apache.seatunnel.api.sink.SeaTunnelSink;
+import org.apache.seatunnel.api.source.SeaTunnelSource;
+import org.apache.seatunnel.api.transform.Transformation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LocalExecutionContext {
+    private SeaTunnelSource source;
```

Comment:
If we have multi source or sink, how implement it?

Reply:
> If we have multi source or sink, how implement it?

I think I need remove `source` and `sink` from LocalExecutionContext. Because we only support define by JobConfigFile now.

Reply:
Also Transformation should be removed